src/utils/configuration.ts

```diff
@@ -32,23 +36,35 @@ const getServerConfigurationValue = <T>(configName: keyof typeof configurationSc
     return defaultValue
 }
 
+const getRunmeTerminalConfiguration = <T>(configName: keyof typeof configurationSchema.terminal, defaultValue: T) => {
+    const configurationSection = workspace.getConfiguration(TERMINAL_SECTION_NAME)
+    const configurationValue = configurationSection.get<T>(configName)!
+    // Since a default value is configured, its safe to use parse here.
+
+    const parseResult = configurationSchema.terminal[configName].safeParse(configurationValue)
```

Comment:
```suggestion

    // Since a default value is configured, its safe to use parse here.
    const parseResult = configurationSchema.terminal[configName].safeParse(configurationValue)
```

Formatting 🤓